src/main/fc/fc_rc.c

```diff
@@ -189,7 +189,7 @@ void processRcCommand(void)
         }
     }
 
-    if (rxConfig()->rcInterpolation || flightModeFlags) {
+    if (rxConfig()->rcInterpolation != RC_SMOOTHING_OFF) {
```

Comment:
How is this change related to BOXHEADADJ? If it isn't, it should go into a separate PR.